### PR TITLE
Fix/indent

### PR DESCRIPTION
**Pull request by maimonav**

### CODE REVIEW

File: packages/editor/web/src/RichContentEditor/handleTabCommand.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  deleteCharacterBeforeCursor,
+  isTypeText,
+  CHARACTERS,
+  getCharacterBeforeSelection,
+} from 'wix-rich-content-editor-common';
+
+const isList = blockType =>
+  blockType === 'ordered-list-item' || blockType === 'unordered-list-item';
+const isCodeBlock = blockType => blockType === 'code-block';
+const getAdjustment = () => (!event.shiftKey ? 1 : -1);
```
Comment (shaulgo) [inline packages/editor/web/src/RichContentEditor/handleTabCommand.js:14]: getDirection

File: packages/editor/web/src/RichContentEditor/handleTabCommand.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+export default (editorState, blockType, customHandlers, command) => {
+  let newState;
+  if (isList(blockType)) {
+    const direction = !event.shiftKey ? 1 : -1;
+    newState = indentSelectedBlocks(editorState, direction);
```
Comment (shaulgo) [inline packages/editor/web/src/RichContentEditor/handleTabCommand.js:20]: getDirection

File: packages/editor/web/src/RichContentEditor/handleBackspaceCommand.js [hunk 3]
```diff
@@ -37,5 +40,14 @@ export default editorState => {
     return EditorState.push(editorState, withoutBlockStyle, 'change-block-type');
   }
 
+  // Last, try to decrease indentation
+  const character = getCharacterBeforeSelection(editorState);
+  if (selection.isCollapsed() && !character) {
```
Comment (shaulgo) [inline packages/editor/web/src/RichContentEditor/handleBackspaceCommand.js:45]:
```suggestion
  if (selection.isCollapsed() && selection.anchorOffset === 0) {
```

Comment (maimonav) [reply packages/editor/web/src/RichContentEditor/handleBackspaceCommand.js:45]: fixed.

File: packages/editor/web/src/RichContentEditor/handleTabCommand.js (hunk 1 above)
Comment (maimonav) [reply packages/editor/web/src/RichContentEditor/handleTabCommand.js:14]: fixed.

File: packages/editor/web/src/RichContentEditor/handleTabCommand.js (hunk 2 above)
Comment (maimonav) [reply packages/editor/web/src/RichContentEditor/handleTabCommand.js:20]: fixed.

File: packages/editor/web/src/RichContentEditor/handleTabCommand.js [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+        newState = insertString(editorState, CHARACTERS.TAB);
+      } else {
+        const character = getCharacterBeforeSelection(editorState);
+        if (character === '\t') {
+          newState = deleteCharacterBeforeCursor(editorState);
+        }
+      }
+    } else {
+      newState = indentSelectedBlocks(editorState, getAdjustment());
+    }
```
[the remark this answers is not in the document]
Comment (maimonav) [reply packages/editor/web/src/RichContentEditor/handleTabCommand.js:34]: fixed.

**Pull request merged**